bucket/tftputil-gui.json

```diff
@@ -0,0 +1,27 @@
+{
+    "version": "1.4.5",
+    "description": "TFTP server and client written in .NET framework.",
+    "homepage": "https://sourceforge.net/projects/tftputil/",
+    "license": "0BSD",
+    "url": "https://dl.sourceforge.net/project/tftputil/TFTPUtil/TFTPUtil%20Version%201.4.5/TFTPUtil_GUI_Version_1.4.5_Binary_Installer.exe#/dl.7z",
+    "hash": "sha1:4f7ecfd0abdd2f7c48438ce64e22981fba6daff4",
+    "post_install": "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\src\", \"$dir\\uninstall.exe\" -Force -Recurse",
+    "bin": [
```

Comment:
Do you think these are good and useful binaries?

Reply:
Remove binaries. They do not have any parameter handling.

Reply:
> Why do you not test installation/uninstallation before posting PR?

Sorry about that. I will pay attention when I apply suggestions.